docs/developer-docs/integrations/sns/integrate-sns/index-integration.md

```diff
@@ -1,16 +1,19 @@
-# Index Canister
+# SNS Index Canister
 
-The index canister keeps track of each transaction stored by the 
-[ledger canister](ledger-integration.md).
+Each SNS is deployed with an index canister that keeps track of each transaction stored by the [ledger canister](ledger-integration.md), sorted by ledger accounts.
```

Comment:
I would phrase this differently "The index canister fetches transactions from the [ledger canister](ledger-integration.md) and indexes them by Account. It allows to query the transactions of an Account in descending order and the list of Account that belongs to a Principal. A index canister is always deployed as part of the SNS canisters. [...]"

Reply:
Updated